lib/mi_bridges/driver/real_properties_page.rb

```diff
@@ -0,0 +1,20 @@
+
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class RealPropertiesPage < BasePage
+      TITLE = "Real Properties"
+
+      def setup; end
+
+      def fill_in_required_fields
+        click_id "assetOther_1"
```

Comment:
maybe we can alias this into a private method so it's clearer what we are clicking on?